scrapy/utils/testproc.py

```diff
@@ -1,13 +1,15 @@
+from __future__ import absolute_import
```

Comment:
just curious about why this is needed and whether this is related to this pull request